.github/workflows/sanity_test.yml

```diff
@@ -46,8 +48,12 @@ jobs:
           python3 -m venv .synapse
           source .synapse/bin/activate
           pip install synapse matrix-synapse
-          curl -sL https://raw.githubusercontent.com/matrix-org/synapse/develop/demo/start.sh --no-rate-limit \
-            | sed s/127.0.0.1/0.0.0.0/g | bash
+          curl -sL https://raw.githubusercontent.com/matrix-org/synapse/develop/demo/start.sh \
+            | sed s/127.0.0.1/0.0.0.0/g | sed 's/http:\/\/localhost/http:\/\/10.0.2.2/g' | bash -s -- --no-rate-limit
```

Comment:
The issue `--no-rate-limit` has already been fixed somewhere, probably in a not-merged PR... Good catch anyway

Reply:
yeahh it was fixed here but it caused 3/4 hour+ runs so I ended up not merging